docs/instructions.md

```diff
@@ -95,15 +95,13 @@ The `examples` directory contains a few simple example agent setups, such as:
     interact with the environment and observe environment mechanics, such as
     rewards or task extras. You will need to install the [PyGame] dependency.
 
-For instance, here is how you can run `run_random_agent.py` in a folder where
-you have your APK file, such as
-the [Apple Flinger](https://github.com/deepmind/android_env/blob/main/docs/example_tasks.md#apple-flinger)
-game.
-(You can find the TAR file with the APK and `.textproto` files
-[here](https://storage.googleapis.com/android_env-tasks/apple_flinger.tar.gz)
-and on the
-[example tasks](https://github.com/deepmind/android_env/blob/main/docs/example_tasks.md)
-page.)
+For instance, here is how you can run
+[`run_random_agent.py`](https://github.com/8bitmp3/android_env/blob/main/examples/run_random_agent.py)
+in a folder where you have your APK file, such as the
```

Comment:
Can you just say "Apple Flinger" instead of "the Apple Flinger game"?

Reply:
Done! Thanks @kenjitoyama